### ISSUE TITLE
Unzel's Field (camp area)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#169]
```
What steps will reproduce the problem?
1. Enter the map area where Unzel is found in the NE corner at his campsite.

What is the expected output? What do you see instead?
A clean, complete, and unscrambled line of trees making up the southern border.
What is actually displayed is quite the opposite.

What version of the product are you using? On what device?
Version 0.6.8 on a Motorola Droid.

Please provide any additional information below.
I believe this issue is a result of a fix made to the tree located at the NW 
corner of the stone or rock formation located in the center of this map area. 
In previous versions, a player could walk through the tree as though it wasn't 
there, which also was a good location to kill off some beetles that were hiding 
there. This issue has been fixed. Now there seems to be a new one.

```

Original issue reported on code.google.com by `ctnbeh13@gmail.com` on 21 Mar 2011 at 10:28

**Comment by GoogleCodeExporter** [comment]
```
Yes, this is a bug. Thank you for reporting it.

Tiled changed the way maps are compressed between versions (from gzip to zlib). 
Fixing the original tree issue caused the maps to be saved with zlib 
compression, and there was a bug in our mapreader that used the zlib inflate 
algorithm.

This is fixed by r57, and should be distributed in the next release.
```

Original comment by `oskar.wi...@gmail.com` on 22 Mar 2011 at 6:16
- Changed state: **Ready**
- Added labels: **Priority-High**, **Milestone-0.6.9**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:51
- Changed state: **Done**
